### ISSUE TITLE
About sampling.py  and  validation dataset

### ISSUE THREAD

**Issue by timchen-2000** [Aboriginer/HFS-SDE#4]
Hi,could you explain how to use ‘ode_sampler’ for sampling? Also, it seems that there are no T1-weighted samples in the knee_multicoil_val_dataset

**Comment by Aboriginer** [comment]
Hi @rookie-Tim-Chen, we do not use ode_sampler in our paper, we only use get_pc_sampler in HFS-SDE. And the T1-weighted test data used in our paper is: file1000046T1.h5, file1000048T1.h5 and file1000049T1.h5.

**Comment by timchen-2000** [comment]
Hi，I have downloaded the dataset as shown in the figure, and didn't find T1-weighted samples in the val_dataset. Did I download the wrong dataset?
![ce79dc9b1b710a1525e8ad812d91add](https://github.com/user-attachments/assets/5b44a49d-e8e0-4e43-8023-bdc7b2215030)

**Comment by Aboriginer** [comment]
Hi @rookie-Tim-Chen, sorry for the late reply. Upon further inspection, We found that we renamed the knee_multicoil_val data to distinguish between T1- and T2-weighted. In fact, the three test samples we used are: file1000976.h5, file1001064.h5, and file1001077.h5 from knee_multicoil_val (approximately 93.8 GB), i.e., 
file1000046T1.h5 is file1000976.h5
file1000048T1.h5 is file1001064.h5
file1000049T1.h5 is file1001077.h5.

We're really sorry about the confusion.
